enables viewers_can_edit: true for grafana

chrisfarms opened the pull request:
I want to be able to see the explore tab, and mess around in grafana.

Since google auth, I (and everyone else) only have the `viewer` role.

viewers_can_edit:true means that anyone with the viewer role can make
transient changes (not saved) which makes it a bit more useful for
poking around without becoming a stateful nightmare.

philandstuff reviewed (dismissed):
needs rebase so this ✅ is worthless to you

Commit:
enables viewers_can_edit: true for grafana  …

I want to be able to see the explore tab, and mess around in grafana.

Since google auth, I (and everyone else) only have the `viewer` role.

viewers_can_edit:true means that anyone with the viewer role can make
transient changes (not saved) which makes it a bit more useful for
poking around without becoming a stateful nightmare.

chrisfarms commented:
reeeeeeebased

Krenair commented:
Actually it's just a case of asking for more rights than viewer.

philandstuff commented:
I still think this is valuable in that it provides a less surprising day-one experience.
